tools/addRollupType/addRollupMainnet8/add_rollup_type_8_output-2024-12-20T17:34:02.808Z.json

```diff
@@ -0,0 +1,24 @@
+ "executeData": "0x134008d30000000000000000000000005132a183e9f3cb7c848b0aac5ae0c4f0491b7ab2000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000144f34eb8eb000000000000000000000000427113ae6f319bffb4459bff96eb8b6bde1a127f000000000000000000000000455ac63e96e6a64ea59c6da0d8f90fca3f1535ab000000000000000000000000000000000000000000000000000000000000000d0000000000000000000000000000000000000000000000000000000000000000e3a7d8bae497945ba8ddc51c69564f60ad4c1a990b9c7bdbd27f7929bfa8f27200000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000060547970653a2056616c696469756d2c2056657273696f6e3a20666f726b2e3133202c2067656e657369733a202f697066732f516d55586e526f5062556d5a75455a43477969486a45736f4e6346567533684c74537668706e664253326d41595500000000000000000000000000000000000000000000000000000000",
+ "id": "0x5b1239cf6d087e5a3a9361e4898853f27820c03456dea6cb59a9803c8415f76e",
+ "decodedScheduleData": {
+  "target": "0x5132A183E9F3CB7C848b0AAC5Ae0c4f0491B7aB2",
+  "value": "0",
+  "data": "0xf34eb8eb000000000000000000000000427113ae6f319bffb4459bff96eb8b6bde1a127f000000000000000000000000455ac63e96e6a64ea59c6da0d8f90fca3f1535ab000000000000000000000000000000000000000000000000000000000000000d0000000000000000000000000000000000000000000000000000000000000000e3a7d8bae497945ba8ddc51c69564f60ad4c1a990b9c7bdbd27f7929bfa8f27200000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000060547970653a2056616c696469756d2c2056657273696f6e3a20666f726b2e3133202c2067656e657369733a202f697066732f516d55586e526f5062556d5a75455a43477969486a45736f4e6346567533684c74537668706e664253326d415955",
+  "decodedData": {
+   "consensusImplementation": "0x427113ae6F319BfFb4459bfF96eb8B6BDe1A127F",
+   "verifier": "0x455ac63E96e6a64EA59C6Da0D8F90FCa3F1535aB",
+   "forkID": "13",
+   "rollupCompatibilityID": "0",
+   "genesis": "0xe3a7d8bae497945ba8ddc51c69564f60ad4c1a990b9c7bdbd27f7929bfa8f272",
+   "description": "Type: Validium, Version: fork.13 , genesis: /ipfs/QmUXnRoPbUmZuEZCGyiHjEsoNcFVu3hLtSvhpnfBS2mAYU"
+  },
+  "predecessor": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "salt": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "delay": "864000"
```

Comment:
is the delay correct!?¿ didint we lower the delay!?¿

Reply:
not yet